
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="author" content="Gregory Moyerbrailean">
    <link rel="icon" href="">

    <title>Research</title>

    <!-- Bootstrap core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <!-- Additional CSS -->
    <link href="css/gmoyerbrailean.css" rel="stylesheet">
  </head>

  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="index.html">gmoyerbrailean</a>
        </div>
        <div class="collapse navbar-collapse">
          <ul class="nav navbar-nav">
          <li><a href="index.html">Home</a></li>
          <li class="active"><a href="research.html">Research</a></li>
          <li><a href="links.html">Links</a></li>
          <li><a href="contact.html">Contact</a></li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </div><!-- /.navbar -->

    <div class="container">
      <div class="starter-template">
        <div id="content">
          <table>
            <tr>
              <td colspan=2><h1>Research</h1></td>
            </tr>
            <tr>
              <td width=200px>
                <ul class="smallind">
                  <li>PhD | Current </li>
                  <li>
                    <ul class="smallind">
                      <li><em>Molec. Biol. and Genetics</em></li>
                      <li><em><a href="http://home.med.wayne.edu/">Wayne State University SOM</a></em></li>
                    </ul>
                  </li>
                  <li>
                  <br>
                  <li>BS | 2010</li>
                  <li>
                    <ul class="smallind">
                      <li><em>Microbiology</em></li>
                      <li><em><a href="http://msu.edu">Michigan State University</a></em></li>
                    </ul>
                  </li>
                  <br>
                  <li><a href="http://scholar.google.com/citations?hl=en&user=WMJw3T8AAAAJ&view_op=list_works&sortby=pubdate">Google Scholar</a></li>
                </ul>
              </td>
              <td>My research focuses on developing computational methods to asses the impact of genetic variants on organismal phenotypes. Our lab attempts to characterize the effect of variants on complex traits by examining changes in chromatin accessability and gene expression surrounding these variants. Through this functional annotation, variants associated with a certain trait or disease (e.g., through GWAS) can be identified as truly causal or not.

                <h2>Projects</h2>
                <div id="projects">
                  <div>
                    <h3 class="inline">Annotation of regulatory variation</h3>
                    
                    <!-- This chunk was me experimenting with a jQuery-based show/hide toggle
                    <div class="btn-group btn-group-xs" data-toggle="buttons">
                      <label class="btn btn-default active" id="anno-show">
                        <input type="radio">Show
                      </label>
                      <label class="btn btn-default" id="anno-hide">
                        <input type="radio">Hide
                      </label>
                    </div> -->
                    
                    <div id="anno">
                    <p>Functional genomics data is increasingly easier and less expensive to generate, especially when coupled with next-gen sequencing technologies. However, understanding the impact of a sequence change within this regions remains a difficult challenge to address. Using the <a href="">CENTIPDE model</a> and our newly developed <a href="">QuASAR model</a>, we have developed an approach to annotating functional regulatory regions and variation within them.</p>
                    <p>We have set up a UCSC browser hub with our annotations, available from the project page here: <a href="http://genome.grid.wayne.edu/centisnps/">http://genome.grid.wayne.edu/centisnps/</a></p>
                    </div>
                  </div>
                </div> <!-- /#projects -->

                <div id="publications">
                  <h2>Publications</h2>
                  <div class="pub">
                    <a href="http://biorxiv.org/content/early/2014/07/29/007559">
                      <p><strong>Moyerbrailean, G. A.</strong>, Harvey, C. T., Kalita, C. A., Wen, X., Luca, F., and Pique-Regi, R. (2014). Are all genetic variants in DNase I sensitivity regions functional?. bioRxiv, 007559.</p>
                    </a>
                  </div>
                  <div class="pub">
                    <a href="http://biorxiv.org/content/early/2014/08/03/007492">
                      <p>Harvey, C., <strong>Moyerbrailean, G. A.</strong>, Davis, O., Wen, X., Luca, F., and Pique-Regi, R. (2014). QuASAR: Quantitative Allele Specific Analysis of Reads. bioRxiv, 007492.</p>
                    </a>
                  </div>
                  <div class="pub">
                    <a href="#">
                      <p>Fujimoto, M., <strong>Moyerbrailean, G. A.</strong>, Noman, S., Gizicki, J. P., Ram, M. L., Henqinet, J., Green, P. A., Ram, J. L. (2014). Application of Ion Torrent sequencing to the assessment of the effect of alkali ballast water treatment on microbial community diversity. PLOS ONE, in review</p>
                    </a>
                  </div>
                  <div class="pub">
                    <a href="http://www.lifescied.org/content/9/3/323.short">
                      <p>Speth, E. B., Momsen, J. L., <strong>Moyerbrailean, G. A.</strong>, Ebert-May, D., Long, T. M., Wyse, S., and Linton, D. (2010). 1, 2, 3, 4: infusing quantitative literacy into introductory biology. CBE-Life Sciences Education, 9(3), 323-332.</p>
                    </a>
                  </div>
                </div><!-- ./publications -->
              </td>
            </tr>
          </table>
        </div>
      </div>
    </div><!-- /.container -->

    <!-- JavaScript -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script src="js/bootstrap.min.js"></script>

    <!-- Google analytics -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-53643157-1', 'auto');
      ga('send', 'pageview');
    </script>
  </body>
</html>
